### PR TITLE
Updated Swedish translation

### PR DESCRIPTION
**Pull request by LarsEJonasson**
Added missing strings

### CODE REVIEW

File: website/public/locales/sv/leaderboard.json.bak [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "accepted": "↪ Accepterad",
```
Comment (andreaskoepf) [inline website/public/locales/sv/leaderboard.json.bak:2]: I guess this `.bak` file was accidentally included and should not be part of this PR.

File: website/public/locales/sv/tasks.json.bak [hunk 2]
```diff
@@ -0,0 +1,92 @@
+{
+  "any_feedback_on_this_task": "Någon feedback på denna uppgift?",
```
Comment (andreaskoepf) [inline website/public/locales/sv/tasks.json.bak:2]: same for this `.bak` file.

Comment (LarsEJonasson) [reply website/public/locales/sv/tasks.json.bak]: Sorry, I deleted them now.

**Review by andreaskoepf — APPROVED** [review]

**Comment by andreaskoepf** [comment]
@LarsEJonasson thanks a lot!
